blocks/education/__info-container/education__info-container.css

```diff
@@ -0,0 +1,30 @@
+.education__info-container{
```

Comment:
I've checked everything is ok. The only one question: don't you want to make this part more responsive that it could change the size with the changing dimensions?

Reply:
You mean, adding calc(...) type of width?